scripts/secnetperf.ps1

```diff
@@ -235,19 +242,29 @@ if ($isWindows) {
 Configure-DumpCollection $Session
 
 # Install any dependent drivers.
-if ($useXDP) { Install-XDP $Session $RemoteDir }
+if ($useXDP -and $isWindows) { Install-XDP $Session $RemoteDir }
 if ($io -eq "wsk") { Install-Kernel $Session $RemoteDir $SecNetPerfDir }
 
 if (!$isWindows) {
     # Make sure the secnetperf binary is executable.
     Write-Host "Updating secnetperf permissions"
+    $GRO = "on"
+    $ENABLE_XDP=0
+    if ($io -eq "xdp") {
+        $GRO = "off"
+        $ENABLE_XDP=1
+    }
     Invoke-Command -Session $Session -ScriptBlock {
         $env:LD_LIBRARY_PATH = "${env:LD_LIBRARY_PATH}:$Using:RemoteDir/$Using:SecNetPerfDir"
+        $env:MSQUIC_ENABLE_XDP=$Using:ENABLE_XDP
         chmod +x "$Using:RemoteDir/$Using:SecNetPerfPath"
+        sudo sh -c "ethtool -K eth0 generic-receive-offload $Using:GRO"
     }
     $fullPath = Repo-Path $SecNetPerfDir
     $env:LD_LIBRARY_PATH = "${env:LD_LIBRARY_PATH}:$fullPath"
+    $env:MSQUIC_ENABLE_XDP=$ENABLE_XDP
     chmod +x "./$SecNetPerfPath"
+    sudo sh -c "ethtool -K eth0 generic-receive-offload $GRO"
```

Comment:
This seems to be failing: https://github.com/microsoft/netperf/actions/runs/8867963327/job/24347253651
```
Error: ethtool: bad command line argument(s)

OriginInfo            : netperf-peer
Exception             : System.Management.Automation.RemoteException: ethtool: 
                        bad command line argument(s)
TargetObject          : ethtool: bad command line argument(s)
CategoryInfo          : NotSpecified: (ethtool: bad command line argument(s):St
                        ring) [], RemoteException
FullyQualifiedErrorId : NativeCommandError
ErrorDetails          : 
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at <ScriptBlock>, <No file>: line 6
PipelineIterationInfo : {}

ethtool: bad command line argument(s)
```